missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java

```diff
@@ -315,17 +315,6 @@ public void onLeave(PlayerArenaLeaveEvent e) {
         }
     }
 
-    @EventHandler
```

Comment:
Why did you remove this? This piece of code should prevent players to take of armor to prevent disguising being from another team, hiding with a special skin, ...